src/base/conv/auto_json.h

```diff
@@ -49,7 +49,10 @@ struct JSON
 	template <class T> inline void primitive(const T &val) const
 	{
 		if constexpr (std::is_arithmetic_v<T>) {
-			json += toString(val);
+			if (std::isfinite(val))
```

Comment:
It's only true for floating-point numbers. Create a new if constexpr/else clause.